modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -4194,7 +4196,8 @@ public void writePropertyBack(String propertyKey, boolean value) {
 
     protected String getContentType(RequestBody requestBody) {
         if (requestBody == null || requestBody.getContent() == null || requestBody.getContent().isEmpty()) {
-            return null;
+            LOGGER.warn("Cannot determine the content type. Default to UNKNOWN_CONTENT_TYPE.");
+            return "UNKNOWN_CONTENT_TYPE";
```

Comment:
Can we fallback to a valid mime type such as `application/octet-stream`?

Reply:
Yup, we can definitely do that but the reason why I did this is to make it consistent with similar unknown cases in the code (e.g. https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L3131) so that users can easily grep "UNKNOWN" to identify which operations/models need to be fixed.

If later users have feedback on having a better default (e.g. application/octet-stream), we can definitely make the switch.